### DISCUSSION TITLE
v6.10.1

### DISCUSSION THREAD

**Discussion by ssorallen** [tabwrangler/tabwrangler#403] [General]
Enhancements:

* [#380] (Firefox only) Use `lastAccessed` time from browser on restart
* [#362] Add Shortcut to lock current tab
* Import many new translations from Crowdin

Issues:

* [#384] Remove custom keyboard shortcut code, both Chrome and Firefox have dedicated interfaces for setting extension shortcuts now

Infrastructure:

* [#374] Auto-sync translations from Crowdin with a GitHub Action
* [9fb4afefa677c6d260931e3197e5a68fa332227e] Convert Tab Wrangler's codebase to TypeScript (remove Flow)

<hr /><em>This discussion was created from the release <a href='https://github.com/tabwrangler/tabwrangler/releases/tag/v6.10.1'>v6.10.1</a>.</em>

**Comment by Pickleplex** [comment]
In multiple Chrome testing (Tab Wrangler) the time close setting has gone in to a 5 sec loop.  Meaning 35 to 31 back to 35 and repeat...  Or 25 to 20 back to 25.  All seconds. I have closed all other windows and disabled all "extensions", still the same.
Please fix this.  Mike

**Reply by ssorallen** [reply]
This means you have more than the minimum number of tabs open so there is nothing to close. The interface needs to be clearer on that, but it sounds like Tab Wrangler is working as intended.
